### PR TITLE
14 css modules stylesheet

### LINKED ISSUE #14: CSS modules stylesheet
https://create-react-app.dev/docs/adding-a-css-modules-stylesheet/

### PR DESCRIPTION
**Pull request by VuBaHung**

### CODE REVIEW

File: apps/frontend/src/app/components/HotelTables/PopupForms/HotelCreateDialog.tsx [hunk 1]
```diff
@@ -142,26 +108,10 @@ export const HotelCreateDialog = ({
...
-                style={{
-                  backgroundColor: '#4CAF50',
-                  color: 'white',
-                  padding: '10px 20px',
-                  border: 'none',
-                  cursor: 'pointer',
-                  borderRadius: '5px',
-                  marginLeft: '10px',
-                }}
+                className={`${styles.create_btn}`}
```
Comment (giapnguyen-innomize) [inline apps/frontend/src/app/components/HotelTables/PopupForms/HotelCreateDialog.tsx]: `className={styles.create_btn}`

File: apps/frontend/src/app/components/HotelTables/PopupForms/HotelUpdateDialog.tsx [hunk 2]
```diff
@@ -1,6 +1,8 @@
 import axios from 'axios';
-import React, { useState } from 'react';
+import { useState } from 'react';
 import { initialState } from '../../../context/HotelProvider';
+import styles from "./hotelUpdateDialog.module.css"
```
Comment (giapnguyen-innomize) [inline apps/frontend/src/app/components/HotelTables/PopupForms/HotelUpdateDialog.tsx]: run nx:format

**Review by giapnguyen-innomize — CHANGES_REQUESTED** [review]
- Rule "Do not commit to unrelated things", I see you have updated the `react-toast`, move it to another pull request
- Sorry about unclear definition, please apply the [sass](https://sass-lang.com/) for this project. the .css file should be .scss
- read this article to learn how to naming the class of css (I will discuss with you later)

**Comment by VuBaHung** [comment]
> * Rule "Do not commit to unrelated things", I see you have updated the `react-toast`, move it to another pull request
> * Sorry about unclear definition, please apply the [sass](https://sass-lang.com/) for this project. the .css file should be .scss
> * read this article to learn how to naming the class of css (I will discuss with you later)

- About `react-toast` it already in the ` validation checking` in previous issue, It notify the response from backend.

**Comment by giapnguyen-innomize** [comment]
> > * Rule "Do not commit to unrelated things", I see you have updated the `react-toast`, move it to another pull request
> > * Sorry about unclear definition, please apply the [sass](https://sass-lang.com/) for this project. the .css file should be .scss
> > * read this article to learn how to naming the class of css (I will discuss with you later)
> 
> * About `react-toast` it already in the ` validation checking` in previous issue, It notify the response from backend.

"in previous issue" mean not this issue so remove this from this pull request
this PR only relation to "CSS modules stylesheet"

**Comment by VuBaHung** [comment]
> > > * Rule "Do not commit to unrelated things", I see you have updated the `react-toast`, move it to another pull request
> > > * Sorry about unclear definition, please apply the [sass](https://sass-lang.com/) for this project. the .css file should be .scss
> > > * read this article to learn how to naming the class of css (I will discuss with you later)
> > 
> > 
> > 
> > * About `react-toast` it already in the ` validation checking` in previous issue, It notify the response from backend.
> 
> "in previous issue" mean not this issue so remove this from this pull request this PR only relation to "CSS modules stylesheet"

Yes, I got it

**Comment by giapnguyen-innomize** [comment]
the Sass not only change the file name to '.scss'
your commit get no value when apply scss.

**Comment by giapnguyen-innomize** [comment]
You need to read BEM naming
- what BEM meaning?
- why we should apply BEM?
- apply to this project

**Pull request merged**